src/main/java/shohei/student/management/controller/converter/StudentConverter.java

```diff
@@ -5,16 +5,16 @@
 import java.util.stream.Collectors;
 import org.springframework.stereotype.Component;
 import shohei.student.management.data.Courses;
-import shohei.student.management.data.Student;
 import shohei.student.management.domain.StudentDetail;
 
 @Component
 public class StudentConverter {
 
-  public List<StudentDetail> convertStudentDetails(List<Student> students,
+  public List<StudentDetail> convertStudentDetails(
+      List<shohei.student.management.data.Student> students,
```

Comment:
これだとうまくimportできませんでしたか？
```suggestion
      List<Student> students,
```

Reply:
修正しました！どこかのタイミングで命名でダブらせてしまっていて、IntelliJのリファクタリング機能を使ったときにこうなってしまっていました。